index.js

```diff
@@ -1,254 +1,380 @@
-            receivedMessage.channel.send("The baking committee draws parallels from baking to algorithmic and programmatic concepts in a unique way.")
-        } else if (arguments == "media"){
-            receivedMessage.channel.send("The media committee is in charge of the BBCS instagram account for trendy lads, and developing and updating the official BBCS20 webite, https://buildingblocs.github.io.")
-        } else if (arguments == "entertainment"){
-            receivedMessage.channel.send("The entertainment committee LOVES playing Dance Dance Revolution. Let's spread the joy by dancing!")
-        } else if (arguments == "games"){
-            receivedMessage.channel.send("Games play games. Are you a gamer? Join and play potato pirates!")
-        } else if (arguments == "lucky"){
-            receivedMessage.channel.send("If you love destroying everyone with your insane luck, join lucky comm and give others a chance at winning prizes ~~or rigging the game to make you win all the time~~!")
-        } else {
-            receivedMessage.channel.send("Sorry what? That ain't a comm. The comms are workshop, coursemology, baking, media, games, lucky, and entertainment.")
-        }
-    }
-    else{
-        receivedMessage.channel.send("Try &committees [committee]. Committees are workshop, coursemology, baking, media, games, lucky, and entertainment.")
+const processCommand = (receivedMessage) => {
+  let fullCommand = receivedMessage.content.substr(1); // Remove the leading &
+  let splitCommand = fullCommand.split(" "); // Split the message up in to pieces for each space
+  let primaryCommand = splitCommand[0]; // The first word directly after the exclamation is the command
+  let arguments = splitCommand.slice(1); // All other words are arguments/parameters/options for the command
+
+  switch (primaryCommand) {
+    case "help":
+      helpCommand(arguments, receivedMessage);
+      break;
+    case "test":
+      testCommand(arguments, receivedMessage);
+      break;
+    case "committee":
+      commCommand(arguments, receivedMessage);
+      break;
+    case "poll":
+      pollCommand(arguments, receivedMessage);
+      break;
+    case "react":
+      reactCommand(arguments, receivedMessage);
+      break;
+    case "confirm":
+      confirmCommand(arguments, receivedMessage);
+      break;
+    case "clear":
+      purgeCommand(arguments, receivedMessage);
+      break;
+    default:
+      receivedMessage.channel.send(
+        "I don't understand the command. Try `&help`"
+      );
+  }
+};
+
+/**
+ * Different roles:
+ * Workshop (June Event) 698841625627197441
+ * Workshop (March Event) 803645952866385940
+ * Workshop (FOSSASIA) 803646041906872370
+ * Quality Assurance 803646515025281034
+ * Emcees 803646503365246986
+ * Resource Management 803646523681406977
+ * Competition 803646520297521172
+ * Entertainment and Lucky Draw 698841626348486737
+ * Games 698841627606777868
+ * Website and Media 698841629867507752
```

Comment:
Are there any security concern for publicly posting these?

Reply:
these role IDs dont unlock any special admin powers so its fine to leave them up for future bot maintainers to refer to.

Reply:
ok! 👍